addGradientInfill.py

```diff
@@ -166,11 +191,10 @@ def main():
                                 segmentLengthRatio * extrusionLength * maxFlow / 100
                             ))
                         else:
-                            splitLine = currentLine.split(" ")
                             outPutLine = ""
                             for element in splitLine:
                                 if "E" in element:
-                                    outPutLine = outPutLine + "E" + str(round(extrusionLength * maxFlow/100,5))
+                                    outPutLine = outPutLine + "E" + str(round(extrusionLength * maxFlow / 100, 5))
```

Comment:
Never forget the spaces around your operators or it can get really confusing. Like reading a 1,2 as a 1.2 etc.